p2p_wallet/UIKit/Views/WLFloatingPanelView.swift

```diff
@@ -17,7 +17,7 @@ class WLFloatingPanelView: BERoundedCornerShadowView {
     
     override func commonInit() {
         super.commonInit()
-        layer.cornerRadius = 12
+        layer.cornerRadius = 8
```

Comment:
Be careful when editing default property of reusable view, this change may affect alot of subviews of `WLFloatingPanelView`, let edit it outside of base class `WLFloatingPanelView`